spot-contracts/contracts/RolloverVault.sol

```diff
@@ -451,9 +450,7 @@ contract RolloverVault is
         IPerpetualTranche perp_ = perp;
         IERC20Upgradeable underlying_ = underlying;
         uint256 underlyingBalPre = underlying_.balanceOf(address(this));
-        (uint256 perpAmtOut, uint256 perpFeeAmtToBurn, SubscriptionParams memory s) = computeUnderlyingToPerpSwapAmt(
-            underlyingAmtIn
-        );
+        (uint256 perpAmtOut, , SubscriptionParams memory s) = computeUnderlyingToPerpSwapAmt(underlyingAmtIn);
```

Comment:
Should we update `computeUnderlyingToPerpSwapAmt` to not return that second argument anymore? It doesn't have a use anymore I don't think.

Reply:
Yea that was my first inclination as well. But considering some cowswap relayers have already integrated the current ABI, I'm leaning towards not rocking the boat.